depfinder/main.py

```diff
@@ -260,9 +260,14 @@ def parse_file(python_file):
         PACKAGE_NAME = os.path.basename(python_file).split('.')[0]
         logger.debug("Setting PACKAGE_NAME global variable to {}"
                      "".format(PACKAGE_NAME))
-    with open(python_file, 'r') as f:
-        code = f.read()
-    catcher = get_imported_libs(code)
+    try:
+        with open(python_file, 'r') as f:
+            code = f.read()
+        catcher = get_imported_libs(code)
+    except SyntaxError:
+        with open(python_file, 'r', encoding='utf-8-sig') as f:
```

Comment:
Can you add an inline note here that this is specifically to catch a problem discovered when analyzing the adal source? After that go ahead and merge and cut another release if you need one